styles/dogs-style.css

```diff
@@ -0,0 +1,75 @@
+section {
+  padding: 1% 15%;
+}
+
+.all-dogs h2 {
+  color: #01344a;
+  text-align: left;
+  font-size: 2rem;
+  font-weight: bold;
+}
+
+.dog-tiles-grid img {
+  width: 100%;
+}
+
+.dog-tiles-grid h3 {
+  text-align: left;
+  padding: 0% 5%;
+  color: #01344a;
+}
+
+.dog-tiles-grid p1 {
+  text-align: left;
+  padding: 0% 5%;
```

Comment:
be careful with using percentages for small padding/margins. Things can go wonky. Rems (or px) are usually a better choice for small things like some of these